wls-ergebnismeldung-service/src/main/java/de/muenchen/oss/wahllokalsystem/ergebnismeldungservice/client/DummyClientImpl.java

```diff
@@ -3,21 +3,46 @@
 import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.configuration.Profiles;
 import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.service.awerte.AWerteClient;
 import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.service.awerte.AWerteModel;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.service.status.sender.StatusClient;
 import de.muenchen.oss.wahllokalsystem.wls.common.exception.WlsException;
 import de.muenchen.oss.wahllokalsystem.wls.common.security.domain.BezirkUndWahlID;
+import java.time.LocalDateTime;
 import java.util.List;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.context.annotation.Profile;
 import org.springframework.stereotype.Component;
 
 @Component
 @Profile(Profiles.DUMMY_CLIENTS)
+@Slf4j
 public class DummyClientImpl
-        implements AWerteClient {
+        implements AWerteClient, StatusClient {
 
     @Override
     public List<AWerteModel> getAWerte(final String wahlbezirkID) throws WlsException {
         return List.of(
                 new AWerteModel(new BezirkUndWahlID("wahlID01", wahlbezirkID), 25L, 26L));
     }
 
+    @Override
+    public void postSchnellmeldungSendungsuhrzeit(BezirkUndWahlID bezirkUndWahlID, LocalDateTime schnellmeldungSendungsuhrzeit)
+            throws WlsException {
+        log.info("postSchnellmeldungSendungsuhrzeit of {} on {}", bezirkUndWahlID, schnellmeldungSendungsuhrzeit);
```

Comment:
In anderen Services wurde explizit im Log erwähnt, dass der DummyClient statt des tatsächlichen Clients aufgerufen wurde. Ich finde das für die spätere Wartung und Fehlerbehebung vorteilhafter, als einfach nur die Parameter auszugeben. Genauso natürlich bei den anderen Methoden
```suggestion
        log.info("Dummy client postSchnellmeldungSendungsuhrzeit of {} on {} called instead of Monitoring", bezirkUndWahlID, schnellmeldungSendungsuhrzeit);
```

Reply:
angepasst mit 8ca3948c40bc62749a8106e09d70c593ba5cc989